src/lib/discovery.ts

```diff
@@ -31,7 +31,7 @@ export interface Resource {
     creationTime: number
 }
 
-export async function startDiscovery(fetch = window.fetch): Promise<boolean> {
+export async function startDiscovery(): Promise<boolean> {
```

Comment:
Is that  `fetch = window.fetch` unnecessary since we can directly access it via `fetch` or what is the reasoning behind getting rid of it in the function parameter section?

Reply:
I dont't know, but VS Code told me, that this is not needed :smile: